### PR TITLE
New filters on product structure API endpoints

### PR DESCRIPTION
**Pull request by meganelepalud-akeneo**

### CODE REVIEW

File: content/updates/2020-08/1-API-search.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# New filters on product structure API endpoints
+::: meta-data type="New" features="API, Productivity" available="late-August" link-to-doc="https://api.akeneo.com/documentation/filter.html#"
+:::
+
+With the new `search` parameter, you can now add filters on product structure endpoints: attributes, attribute groups, categories, and families.
```
Comment (NolwennP) [inline content/updates/2020-08/1-API-search.md:13]:
```suggestion
With the new `search` parameter, you can now add filters on product structure endpoints: attributes, attribute groups, categories, and families.  
```
Two spaces at the end of the line to force going to the next line.

File: content/updates/2020-08/1-API-search.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+With the new `search` parameter, you can now add filters on product structure endpoints: attributes, attribute groups, categories, and families.  
+For example, you can retrieve data for all attributes that have been updated since a specific date, or you can get the data from all the sub-categories of a specific tree.
+
+::: more
+[How to filter attributes?](https://api.akeneo.com/documentation/filter.html#filter-attributes)
+[How to filter attribute groups?](https://api.akeneo.com/documentation/filter.html#filter-attribute-groups)
+[How to filter families?](https://api.akeneo.com/documentation/filter.html#filter-families)
+[How to filter categories?](https://api.akeneo.com/documentation/filter.html#filter-categories)
+[How to manage your connections?](../articles/manage-your-connections.html)
```
Comment (NolwennP) [inline content/updates/2020-08/1-API-search.md:21]: As you don't mention the connections at all in the news, it feels weird to have a link to the documentation about connections.

File: content/updates/2020-08/1-API-search.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+pim_announcement_img: ./img/api-filter-attributes.png
+pim_announcement_alt_img: "API documentation about attribute filters"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# New filters on product structure API endpoints
+::: meta-data type="New" features="API, Productivity" available="late-August" link-to-doc="https://api.akeneo.com/documentation/filter.html#"
```
Comment (NolwennP) [inline content/updates/2020-08/1-API-search.md:10]:
```suggestion
::: meta-data type="New" features="Connectivity" available="late-August" link-to-doc="https://api.akeneo.com/documentation/filter.html#"
```
@LaureBro what do you think?

Comment (NolwennP) [inline content/updates/2020-08/1-API-search.md:10]:
```suggestion
::: meta-data type="New" features="API, Productivity" available="late August" link-to-doc="https://api.akeneo.com/documentation/filter.html"
```

File: content/updates/2020-08/1-API-search.md (hunk 2 above)
Comment (meganelepalud-akeneo) [reply content/updates/2020-08/1-API-search.md:21]: I hesitated when I added the link about connections but you're right.

File: content/updates/2020-08/1-API-search.md (hunk 3 above)
Comment (meganelepalud-akeneo) [reply content/updates/2020-08/1-API-search.md]: I'd still let the API part.

**Review by NolwennP — COMMENTED** [review]
For the screenshot, can you take it without the shadow at the top?

**Pull request closed without merging**